bluezero/advertisement.py

```diff
@@ -123,9 +123,13 @@ def service_UUIDs(self, UUID):  # pylint: disable=invalid-name
                  'ServiceUUIDs',
                  UUID)
 
-    def manufacturer_data(self):
-        """Manufacturer Data to be broadcast (Currently not supported)"""
-        pass
+    def manufacturer_data(self, company_id, data):
+        """Manufacturer Data to be broadcast
+
+        ad.manufacturer_data({int("0x004c", 16):bytearray.fromhex(23*"AA")})"""
```

Comment:
This doesn't feel list a very helpful docstring as I wouldn't expect anyone to assign the values that way.
Can we also not use the Apple Inc Company Identifier of 0x004c.

Reply:
yes i removed this. I hope with the example it should be clear